dist/js/main.js

```diff
@@ -1,68 +1,100 @@
-// getting html elements
+// Getting html elements
 const bookContainer = document.querySelector('.book-list__container');
 const title = document.querySelector('#book-title');
 const author = document.querySelector('#book-author');
 const submitBtn = document.querySelector('#book-submit');
 
-const bookLibrary = [];
+// Creating book and library classes
+class Library {
+  constructor(books = []) {
+    this.books = books;
+  }
+
+  add(book) {
+    this.books.push(book);
+  }
+
+  remove(index) {
+    this.books.splice(this.getBook(index), 1);
```

Comment:
[OPTIONAL] It would be better to use the [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter)() method that doesn't mutate the array instead of the splice method